data/json/construction.json

```diff
@@ -1510,6 +1510,27 @@
     "pre_special": "check_empty",
     "post_terrain": "f_bed"
   },
+  {
+    "type": "construction",
+    "description": "Build Bed Frame",
+    "category": "FURN",
+    "required_skills": [ [ "fabrication", 4 ] ],
+    "time": 55,
+    "qualities": [ [ { "id": "HAMMER", "level": 2 } ], [ { "id": "SAW_W", "level": 1 } ] ],
+    "components": [ [ [ "2x4", 12 ] ], [ [ "nail", 10 ] ] ],
+    "pre_special": "check_empty",
+    "post_terrain": "f_bed_frame"
+  },
+  {
+    "type": "construction",
+    "description": "Add Mattress to Bed Frame",
+    "category": "FURN",
+    "required_skills": [ [ "fabrication", 0 ] ],
+    "time": 5,
+    "components": [ [ "mattress: ] ],
```

Comment:
This line looks like it's missing a few things

Reply:
Is that from an older merge?

Reply:
Looks so, my bad.